deebot_client/events/water_info.py

```diff
@@ -17,6 +17,13 @@ class WaterAmount(IntEnum):
     HIGH = 3
     ULTRAHIGH = 4
 
+@unique
+class SweepType(IntEnum):
+    """Enum class for all possible sweeping types."""
+
+    Standard = 1
+    Deep = 2
```

Comment:
```suggestion
    STANDARD = 1
    DEEP = 2
```

Please use the Python naming convetion